openapi/schemas/Configuration/Configuration.json

```diff
@@ -0,0 +1,180 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Propel Configuration API",
+    "description": "Propel Configuration API",
+    "license": {
+      "name": "MIT",
+      "url": "https://spdx.org/licenses/MIT"
+    },
+    "contact": {
+      "name": "Propel Support",
+      "url": "https://propel.force.com/helpcenter/s/",
+      "email": "support@propelsoftware.com"
+    }
+  },
+  "servers": [
+    {
+      "url": "https://{endpoint}/services/apexrest/{namespace}",
+      "description": "Propel API Endpoint Base URL",
+      "variables": {
+        "endpoint": {
+          "default": "login.salesforce.com",
+          "description": "This is the Salesforce instance url"
+        },
+        "namespace": {
+          "default": "PDLM",
+          "description": "This value should default to PDLM for managed package environments. For other environments, provide the namespace to your dev org."
+        }
+      }
+    }
+  ],
+  "paths": {
+    "/configuration/": {
+      "post": {
```

Comment:
question: per your definition file, this API only supports POST operation. is that accurate, or does your API also support GET to retrieve custom setting/metadata? or PATCH to update custom setting/metadata?

Reply:
yeah strictly speaking PATCH should be the one that does the update while POST does the creation. However the API currently only supports POST to do everything. When the API is refined then the spec here needs to be updated to reflect that but for now this spec is accurate of what the API supports.

Reply:
Great, thank you for the confirmation!